lib/OpenQA/Controller/Main.pm

```diff
@@ -67,13 +67,16 @@ sub index {
                         $jr{failed}++;
                         next;
                     }
+                    if ( grep { $job->state eq $_ } OpenQA::Schema::Result::Jobs::INCOMPLETE_RESULTS ) {
```

Comment:
I don't get it. There is already check that $job->state is eq DONE. Aren't you trying to check for $job->result instead?